pkg/bucket/bandwidth/throttle.go

```diff
@@ -0,0 +1,107 @@
+/*
+ * MinIO Cloud Storage, (C) 2020 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package bandwidth
+
+import (
+	"context"
+	"sync"
+	"sync/atomic"
+	"time"
+)
+
+const (
+	millisecondsInSecond   = time.Second / time.Millisecond
+	intervalInMilliseconds = 250
```

Comment:
I would prefer this to be typed instead of having it in the name.

It represents a duration, so it should be a duration.

```suggestion
	throttleInterval = 250 * time.Millisecond.
```

Reply:
Will make the change and for the ones below as well